docs/querying/api.md

```diff
@@ -363,6 +363,98 @@ $ curl http://localhost:9090/api/v1/targets
 }
 ```
 
+
+## Rules
+
+The `/rules` API endpoint returns a list of alerting and recording rules that
+are currently loaded. In addition it returns the currently active alerts fired
+by the Prometheus instance of each alerting rule.
+
+As the `/rules` endpoint is fairly new, it does not have the same stability
+guarantees as the overarching API v1.
+
+```
+GET /api/v1/rules
+```
+
+```json
+$ curl http://localhost:9090/api/v1/rules
+
+{
+    "data": {
+        "groups": [
+            {
+                "rules": [
+                    {
+                        "alerts": [
+                            {
+                                "activeAt": "2018-07-04T20:27:12.60602144+02:00",
+                                "annotations": {},
+                                "labels": {
+                                    "alertname": "my-alert"
+                                },
+                                "state": "firing",
+                                "value": 1
+                            }
+                        ],
+                        "annotations": {},
+                        "duration": 0,
+                        "labels": {},
+                        "name": "my-alerting-rule",
+                        "query": "vector(1)",
+                        "type": "alerting"
+
+                    },
+                    {
+                        "labels": {},
```

Comment:
Should we have an omitempty on this? Labels aren't typical on recording rules, and having this here could be a bit confusing.